src/guess_density.jl

```diff
@@ -109,7 +109,8 @@ function gaussian_superposition(basis::PlaneWaveBasis{T}, gaussians) where {T}
     end
 
     # projection in the normalized plane wave basis
-    irfft(basis, ρ / sqrt(basis.model.unit_cell_volume))
+    # (ignore real check here as fft-grid can contain G-vectors without their -G counterpart)
+    irfft(basis, ρ / sqrt(basis.model.unit_cell_volume), check=Val(false))
```

Comment:
force_real is quite expensive. This is just the guess. Accuracy does not matter so much here. But it might make sense to put that in at other places where I used check=false.

Reply:
Indeed can be avoided rather easily, actually.